src/github.com/getlantern/flashlight/logging/logging_test.go

```diff
@@ -0,0 +1,36 @@
+package logging
+
+import (
+	"bytes"
+	"encoding/json"
+	"strings"
+	"testing"
+
+	"github.com/getlantern/go-loggly"
+	"github.com/getlantern/golog"
+	"github.com/getlantern/testify/assert"
+)
+
+func TestLoggly(t *testing.T) {
+	var buf bytes.Buffer
+	var result map[string]interface{}
+	loggly := loggly.New("token not required")
+	loggly.Writer = &buf
+	lw := logglyErrorWriter{client: loggly}
+	golog.SetOutputs(lw, nil)
+	log := golog.LoggerFor("test")
+
+	log.Error("short message")
+	if assert.NoError(t, json.Unmarshal(buf.Bytes(), &result), "Unmarshal error") {
+		assert.Equal(t, "test", result["locationInfo"])
+		assert.Equal(t, "test: short message\n", result["message"])
+	}
+
+	buf.Reset()
+	longMsg := "really l" + strings.Repeat("o", 100) + "ng message"
+	log.Error(longMsg)
+	if assert.NoError(t, json.Unmarshal(buf.Bytes(), &result), "Unmarshal error") {
+		assert.Equal(t, "test", result["locationInfo"])
+		assert.Equal(t, ("test: " + longMsg)[0:100], result["message"], "message should be truncated to 100 characters")
+	}
```

Comment:
I like how you're testing through the loggly API!  I thought of a couple more things that are probably good to do:

1. Add this test to `testpackages.txt`

2. I think we can use better logic for really long messages.  

Take for example this error: `Mar 09 22:00:27.900 - enproxy: Dest: ui.getlantern.org:443    ProxyHost: fl-nl-20150226-007.getiantem.org    Unable to issue read request: Error reading response from proxy: unexpected EOF`.

Your logic takes the first 100 bytes, which is `enproxy: Dest: ui.getlantern.org:443    ProxyHost: fl-nl-20150226-007.getiantem.org    Unable to is`

It would also be useful to take chunks from the tail, terminating at the colons.  For the example, this would be:

```
unexpected EOF
Error reading response from proxy: unexpected EOF
Unable to issue read request: Error reading response from proxy: unexpected EOF
```

These would be very nice for grouping since they're complete messages and will often match similar messages.